Source/Scene/Model.js

```diff
@@ -1388,7 +1429,35 @@ define([
         }
 
         var bufferView;
+        var accessors = model.gltf.accessors;
         var bufferViews = model.gltf.bufferViews;
+
+        var accessorsForBufferViewId = {};
```

Comment:
Probably put this in its own function that returns `accessorsForBufferViewId`.  See `Statements in a function should be at a similar level of abstraction...` [here](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#functions).